lib/daru/io/importers/excel.rb

```diff
@@ -4,19 +4,29 @@ module Daru
   module IO
     module Importers
       class Excel < Base
-        Daru::DataFrame.register_io_module :from_excel, self
+        Daru::DataFrame.register_io_module :from_excel do |*args|
+          if args.first.end_with? '.xlsx'
+            require 'daru/io/importers/excelx'
```

Comment:
Yes, a `require` will be called each time the `from_excel` method is called with a `.xlsx` file. But I'm not sure why it'd be inefficient - isn't `require` by default done only once?  And would this make it better?
```ruby
require 'daru/io/importers/excelx' unless defined?(Daru::IO::Importers::Excelx)
```

> perform all the requires when the gem is required or when a specific importer/exporter is required.

When a user's use-case is only `.xls` files, the excelx importer isn't needed to be `require`d right? (Or is unconditionally requiring excelx importer when requiring excel importer, a better standard?)